akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -184,11 +184,18 @@ Distributed Data Mode
 
 Instead of using :ref:`persistence-java` it is possible to use the :ref:`distributed_data_java` module
 as storage for the state of the sharding coordinator. In such case the state of the 
-``ShardCoordinator`` will be replicated inside a cluster by the Distributed Data module with
+``ShardCoordinator`` will be replicated inside a cluster by the :ref:`distributed_data_java` module with
 ``WriteMajority``/``ReadMajority`` consistency.
 
-This mode can be enabled by setting configuration property ``akka.cluster.sharding.state-store-mode``
-as ``ddata``. 
+This mode can be enabled by setting configuration property::
+
+    akka.cluster.sharding.state-store-mode=ddata 
```

Comment:
nitpick: I'd personally like spaces around the setting, makes it easier to parse where key ends.